### DISCUSSION TITLE
11.0.0-a2

### DISCUSSION THREAD

**Discussion by SubJunk** [UniversalMediaServer/UniversalMediaServer#2964] [General]
Thanks for your feedback on the first alpha release! Here is a minor update to improve stability.

Changes since 11.0.0-a1

	General:
		Backgrounds for TV series, episodes and movies fade in
		Image backgrounds in Media Library without text are prioritized over ones with text
		Match more TV series in API
		All v10 changes up to 10.21.1

<hr /><em>This discussion was created from the release <a href='https://github.com/UniversalMediaServer/UniversalMediaServer/releases/tag/11.0.0-a2'>11.0.0-a2</a>.</em>

**Comment by ik666** [comment]
I've just checked this release today. Very nice work. My impression is a noticeable speed improvement.

**Reply by SubJunk** [reply]
Thanks for the feedback! It's really helpful

**Comment by schroef** [comment]
I tried the update version, from 10.21.0.1, after download and restart, i still have the same version?!

**Reply by SubJunk** [reply]
The latest stable public version is 10.21.0.1

**Reply by schroef** [reply]
Ow okay, i thought the number shown in the header here represented the release version

**Reply by SubJunk** [reply]
Ah yeah, I see. The `-a2` part stands for `alpha 2`, and alpha means pre-release, feature-incomplete. It is quite stable though and I recommend using it. You can get it by clicking "Download beta" on the website download section.

**Reply by schroef** [reply]
Installing it, still showed v 10.21.0.1 on Windows, is that correct?

**Reply by SubJunk** [reply]
No, if you install v11 it should show as v11. Today we are releasing v11.0.0 to the auto updater so hopefully that works for you

**Comment by ralvarador** [comment]
System:
* Laptop
* CPU - AMD Ryzen 5 2500U with Radeon Vega Mobile Gfx - 4 Cores
* RAM - 16 GB
* GPU - APU - AMD Radeon(TM) Vega 8 Graphics - Main/Integrated, VRAM - 1024 MB - DDR4 1200 MHz
   Driver Version - 22.10.17.01-220517a-379487E-AMD-Software-Adrenalin-Edition
* OS Windows 10 Professional (64 bit) - 21H2

I had an issue: If an specific NIC is selected in the "Force Networking on Interface" dropdown, UMS was not able to use the selected NIC. To make things work again, I had to select the first option available in the dropdown (blank option (? ):

_From "" (blank) -> selecting the Laptop's WiFi NIC:_
**02:02:37 DEBUG Reading network_interface: "Intel(R) Dual Band Wireless-AC 8265" (previous: "", default: "")**
02:02:37 INFO  Using forced network interface: Intel(R) Dual Band Wireless-AC 8265
**02:02:37 ERROR Forced network interface Intel(R) Dual Band Wireless-AC 8265 not found on this system**
02:02:37 INFO  Network not available with the config values.
02:02:37 INFO  Waiting network scanner discovering.
02:02:37 INFO  Using forced network interface: Intel(R) Dual Band Wireless-AC 8265
**02:02:37 ERROR Forced network interface Intel(R) Dual Band Wireless-AC 8265 not found on this system**
02:02:37 DEBUG task ended

_Switching back to the "" (blank option):_
**02:07:15 DEBUG Reading network_interface: "" (previous: "Intel(R) Dual Band Wireless-AC 8265", default: "")**
02:07:15 DEBUG Starting UPnP (JUPnP) services.
02:07:15 DEBUG Upnp set in silent log mode
**02:07:15 DEBUG UPnP (JUPnP) services are online, listening for media renderers**
02:07:15 DEBUG task ended

_Switching again to the Laptop's WiFi NIC:_
**02:08:08 DEBUG Reading network_interface: "Intel(R) Dual Band Wireless-AC 8265" (previous: "", default: "")**
02:08:08 INFO  Using forced network interface: Intel(R) Dual Band Wireless-AC 8265
02:08:08 ERROR Forced network interface Intel(R) Dual Band Wireless-AC 8265 not found on this system
02:08:08 INFO  Network not available with the config values.
02:08:08 INFO  Waiting network scanner discovering.
02:08:08 INFO  Using forced network interface: Intel(R) Dual Band Wireless-AC 8265
**02:08:08 ERROR Forced network interface Intel(R) Dual Band Wireless-AC 8265 not found on this system**
02:08:08 DEBUG task ended

_Finally set the option to "" (blank) - again... and things working again_
**02:12:46 DEBUG Reading network_interface: "" (previous: "Intel(R) Dual Band Wireless-AC 8265", default: "")**
02:12:46 DEBUG Starting UPnP (JUPnP) services.
02:12:46 DEBUG Upnp set in silent log mode
**02:12:46 DEBUG UPnP (JUPnP) services are online, listening for media renderers**
02:12:46 DEBUG task ended

**Reply by SubJunk** [reply]
Thanks for reporting this. Any ideas @SurfaceS ?

**Comment by Mik-S-UMS** [comment]
@SubJunk could you have a look at [this ](https://www.universalmediaserver.com/forum/viewtopic.php?f=4&t=15147), folders are appearing empty on his renderer in this version.

**Reply by SubJunk** [reply]
@Mik-S-UMS thanks for letting me know. I will try to have a look. Any ideas @SurfaceS ?

**Comment by ik666** [comment]
@Mik-S-UMS Maybe it is fixed in 11.1.0 with PR #2897

**Reply by Mik-S-UMS** [reply]
I'll let him know to wait for the release of that version to everyone and in the meantime he can check to see if it happens in 10.21 too

**Comment by SurfaceS** [comment]
Maybe he can try the 10.21 to confirm the bug is in it since 10.19.
I don't think it's related to v11 things.
https://www.fosshub.com/Universal-Media-Server-old.html?dwl=UMS-10.21.1.exe

**Reply by Mik-S-UMS** [reply]
He tried a few versions and found the bug first appeared in 10.20.0 so not related to v11 stuff but the bug had been carried forward.
He has uploaded logs for 10.19.0 10.20.0 and 10.21.0 on the forum if they are helpful.
It might be a good idea to send him a link to 11.1.0 to see if it was #2897 that is causing it.

**Reply by SubJunk** [reply]
#2897 was reported before 10.20.0 was released so it wouldn't be the same bug.
That's great they have narrowed it down to that version though.

**Reply by Mik-S-UMS** [reply]
He has reported that 11.1.0 is the same so it was not that bug.

**Reply by ik666** [reply]
Maybe he can delete his database just in case some entries are corrupt ?!

**Reply by Mik-S-UMS** [reply]
He tried that. It was one of the first things I suggested too.

**Reply by SubJunk** [reply]
I'll have a look at the logs soon, been busy this week

**Reply by SubJunk** [reply]
One thing that could explain it was the addition of `dc:date` in the results, even though we seem to be doing it correctly. I've posted a test build

**Reply by SubJunk** [reply]
That was the cause of it
